.github/workflows/nightly_ondemand_whl.yml

```diff
@@ -0,0 +1,365 @@
+name: Nightly-OnDemand WHL Tests
```

Comment:
Let's rename it as `Torch Nightly WHL Tests`

Reply:
Do you mean we just need keep nightly and remove all on-demand?